src/applications/toe/tests/fixtures/data/form-submission-test-data.js

```diff
@@ -66,7 +66,6 @@ export const submissionForm = {
     },
   },
   reviewPageView: {
-    openChapters: [],
```

Comment:
You will see lots of code in this PR removed that has to do with open chapters. Redux code, fixture code, actions, etc. All of this code supported keeping track of and firing events on the review page chapter accordions. Since we're migrating to the `<va-accordion>` which handles its own state and events, we can take it all out.